index.bs

```diff
@@ -725,10 +730,12 @@ The <dfn attribute for="XRReferenceSpace">onreset</dfn> attribute is an [=Event
 
 When an {{XRReferenceSpace}} is requested, the user agent MUST <dfn>create a reference space</dfn> by running the following steps:
 
-  1. Initialize [=XRSpace/session=] be the {{XRSession}} object that requested creation of a reference space.
+  1. Let |session| be the {{XRSession}} object that requested creation of a reference space.
+  1. Initialize [=XRSpace/session=] to be |session|.
   1. Let |options| be the {{XRReferenceSpaceOptions}} passed to {{requestReferenceSpace()}}.
   1. Let |type| be set to |options| {{XRReferenceSpaceOptions/type}}.
   1. Let |referenceSpace| be set to <code>null</code>.
+  1. If |type| is {{identity}} let |referenceSpace| be a new {{XRReferenceSpace}}.
```

Comment:
We don't really need to, in my opinion. If you passed in any other non-applicable values in the dictionary (ie: `{ type: 'identity', foo: 'bar' }`) then we just ignore it, so that feels like it could carry over to this as well? Then again, if you passed a subtype that wasn't valid the IDL would cause it to naturally error whether or not the type uses it, so maybe being more rigorous about it isn't a bad idea? IDK.

Reply:
My general opinion is that we should do what's consistent with the rest of the web on this one.  Got an API ergonomics person who wants to give us guidance one way or another?

Reply:
I'll poke around and see.